docs/developers/releasing.md

```diff
@@ -40,7 +40,16 @@ The highlighted updates, for example, new key features. This part is collected m
 ### Instrumentation
 * Observability-related changes. For example, metric adding/event recording.
 ```
-Compare the newly created minor release branch with the previous minor release tag, for example, compare `release-1.4` branch with `v1.3.0` tag, to get all relevant changes. Then extract the preceding types of release notes from these changes. For example, extract [this](https://github.com/karmada-io/karmada/pull/2675) as:
+#### Collect changes(manually)
+Compare the newly created minor release branch with its pre-release tag, for example, compare `release-1.8` branch with `v1.8.0-alpha.0` tag, to get all relevant changes.  Here's how it works on GitHub:
+
+- Enter the [Compare page](https://github.com/karmada-io/karmada/compare).
+- Choose the pre-release tag as the base ref.
+- Choose the newly created minor release branch as the head ref.
```

Comment:
But the picture is wrong. In the picture, the left side should be the head, and the right should be  pre-release.

Reply:
![image](https://github.com/karmada-io/website/assets/81208264/1a2b8581-b006-4755-9571-411a9f8a2d78)
The left side represent the base ref, the right side represent the head ref.

Reply:
Oh, you are right!